packages/fx-core/src/common/tools.ts

```diff
@@ -493,3 +495,27 @@ export function isFeatureFlagEnabled(featureFlagName: string): boolean {
   }
   return false;
 }
+
+export function generateBicepFiles(
+  templateFilePath: string,
+  context: any
+): Result<string, FxError> {
+  try {
+    const templateString = fs.readFileSync(templateFilePath, "utf8");
```

Comment:
Is it possible to change it to its non-blocking counterpart?

Reply:
Yes. Let me fix that.

Reply:
Fixed.